test/Dialect/SV/basic.mlir

```diff
@@ -185,8 +185,8 @@ hw.module @test1(%arg0: i1, %arg1: i1, %arg8: i8) {
     }
   }
 
-  // CHECK-NEXT: %combWire = sv.reg : !hw.inout<i1>
-  %combWire = sv.reg : !hw.inout<i1>
+  // CHECK-NEXT: %combWire = sv.reg svattrs [#sv.attribute<"dont_merge">] : !hw.inout<i1>
+  %combWire = sv.reg svattrs [#sv.attribute<"dont_merge">] : !hw.inout<i1>
```

Comment:
It would be good to also check the `const_expression` variant here even though it is already covered by the `sv-dialect.mlir` test.